pkg/server/container_stop.go

```diff
@@ -65,22 +65,24 @@ func (c *criContainerdService) StopContainer(ctx context.Context, r *runtime.Sto
 		return &runtime.StopContainerResponse{}, nil
 	}
 
-	// TODO(random-liu): [P1] Get stop signal from image config.
-	stopSignal := unix.SIGTERM
-	glog.V(2).Infof("Stop container %q with signal %v", id, stopSignal)
-	_, err = c.containerService.Kill(ctx, &execution.KillRequest{ID: id, Signal: uint32(stopSignal)})
-	if err != nil {
-		if isContainerdContainerNotExistError(err) {
-			return &runtime.StopContainerResponse{}, nil
+	if r.GetTimeout() > 0 {
+		// TODO(random-liu): [P1] Get stop signal from image config.
+		stopSignal := unix.SIGTERM
```

Comment:
Please comment below that calling c.containerService.Delete results in a SIGKILL being sent